<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>855</width>
    <height>520</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="btn_browse1">
   <property name="geometry">
    <rect>
     <x>670</x>
     <y>83</y>
     <width>141</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>17</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Browse</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_input">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>57</y>
     <width>191</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>17</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>Input File Path:</string>
   </property>
  </widget>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>26</x>
     <y>40</y>
     <width>801</width>
     <height>16</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="Line" name="line_2">
   <property name="geometry">
    <rect>
     <x>26</x>
     <y>490</y>
     <width>801</width>
     <height>16</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QProgressBar" name="pgbar_download">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>380</y>
     <width>771</width>
     <height>23</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="value">
    <number>0</number>
   </property>
   <property name="textVisible">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_ouput">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>137</y>
     <width>211</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>17</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>Output File Path:</string>
   </property>
  </widget>
  <widget class="Line" name="line_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>47</y>
     <width>31</width>
     <height>451</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="Line" name="line_4">
   <property name="geometry">
    <rect>
     <x>820</x>
     <y>47</y>
     <width>16</width>
     <height>451</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_output">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>167</y>
     <width>611</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_execute">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>310</y>
     <width>771</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>17</pointsize>
    </font>
   </property>
   <property name="text">
    <string> L3 Area Measure</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_input">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>87</y>
     <width>611</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>        &quot;Drag &amp; Drop Folder here&quot; OR &quot;Click Browse button&quot;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_browse2">
   <property name="geometry">
    <rect>
     <x>670</x>
     <y>160</y>
     <width>141</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>17</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Browse</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_upload">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>93</y>
     <width>21</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_logo">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>10</y>
     <width>231</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_execute">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>230</y>
     <width>41</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="Line" name="line_5">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>280</y>
     <width>771</width>
     <height>21</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="Line" name="line_6">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>420</y>
     <width>771</width>
     <height>21</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_num_input">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>450</y>
     <width>251</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>17</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>No. Input File:</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_num_output">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>450</y>
     <width>271</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>17</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>No. Output File:</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_time">
   <property name="geometry">
    <rect>
     <x>630</x>
     <y>450</y>
     <width>171</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>17</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>Time: </string>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_batch">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>233</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>17</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>Batch Size</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_16">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>230</y>
     <width>71</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>17</pointsize>
    </font>
   </property>
   <property name="text">
    <string>16</string>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_32">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>230</y>
     <width>71</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>17</pointsize>
    </font>
   </property>
   <property name="text">
    <string>32</string>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_64">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>230</y>
     <width>71</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>17</pointsize>
    </font>
   </property>
   <property name="text">
    <string>64</string>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_128">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>230</y>
     <width>71</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>17</pointsize>
    </font>
   </property>
   <property name="text">
    <string>128</string>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
